app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ServerViewModel.kt

```diff
@@ -2,33 +2,43 @@ package co.netguru.baby.monitor.client.feature.server
 
 import androidx.lifecycle.*
 import co.netguru.baby.monitor.client.common.ISchedulersProvider
+import co.netguru.baby.monitor.client.common.SingleLiveEvent
 import co.netguru.baby.monitor.client.data.DataRepository
+import co.netguru.baby.monitor.client.data.communication.websocket.ClientConnectionStatus
 import co.netguru.baby.monitor.client.data.server.CameraState
 import co.netguru.baby.monitor.client.data.splash.AppState
 import co.netguru.baby.monitor.client.feature.communication.nsd.NsdServiceManager
 import co.netguru.baby.monitor.client.feature.communication.webrtc.RtcConnectionState
 import co.netguru.baby.monitor.client.feature.communication.webrtc.server.WebRtcService
+import co.netguru.baby.monitor.client.feature.communication.websocket.Message
+import co.netguru.baby.monitor.client.feature.communication.websocket.Message.Companion.RESET_ACTION
+import co.netguru.baby.monitor.client.feature.communication.websocket.WebSocketServerService
+import dagger.Lazy
 import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.disposables.Disposable
 import io.reactivex.rxkotlin.addTo
 import io.reactivex.rxkotlin.plusAssign
 import io.reactivex.rxkotlin.subscribeBy
+import io.reactivex.schedulers.Schedulers
 import timber.log.Timber
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 
 class ServerViewModel @Inject constructor(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/b1aa35793dca6d7714e5534d6167592f0bbe6a99/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ServerViewModel.kt#L29"></span>Class ‘ServerViewModel’ with ‘15’ functions detected. Defined threshold inside classes is set to ‘15’</td>
    </tr>
  </tbody>
</table>